mappings/net/minecraft/world/dimension/NoiseSamplingConfig.mapping

```diff
@@ -0,0 +1,15 @@
+CLASS net/minecraft/class_5308 net/minecraft/world/dimension/NoiseSamplingConfig
+	FIELD field_24799 CODEC Lcom/mojang/serialization/Codec;
+	FIELD field_24800 xzScale D
+	FIELD field_24801 yScale D
+	FIELD field_24802 xzFactor D
+	FIELD field_24803 yFactor D
```

Comment:
imo horizontal/vertical factor/scale would be better

Reply:
They're called like this in the config files. https://gist.github.com/RainWarrior/2c32848aad78910a3ca148b82c91f7c2#file-worldgen_settings-json-L204

Reply:
Yeah, I'm just basing this off the codecs.